e2-studio-star-rx72n-firmware/libs/rx_usb_comm/src/rx_usb_comm.c

```
@@ -1257,18 +1165,76 @@ rx_usb_comm_receive(rx_usb_comm_handle_t* handle, rx_frame_t* frame, const uint3
   rx_err_t            err        = k_rx_ok;
   rx_receive_result_t result     = k_receive_continue;
 
-  /* Try to receive a complete frame with bounded iterations */
+  /* Try to receive a complete frame with bounded iterations.
+   * Control frames (PING, PONG, RESET, RESET_ACK) are handled internally
+   * and the loop continues to receive the next data frame. */
   while (iterations < k_max_receive_iterations) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)